src/Workspaces/CoreTest/ServicesTest.csproj

```diff
@@ -10,7 +10,6 @@
     <RootNamespace>Microsoft.CodeAnalysis.UnitTests</RootNamespace>
     <AssemblyName>Roslyn.Services.UnitTests</AssemblyName>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
-    <GenerateMSBuildBindingRedirects>true</GenerateMSBuildBindingRedirects>
```

Comment:
Our target files add a reference to ```build/Defaults/Desktop/app.config``` to all test projects, which already has redirects.